scripts/generateWebApiIndex.js

```diff
@@ -0,0 +1,58 @@
+const { XMLParser } = require("fast-xml-parser");
+const { get } = require("axios");
+const { writeFileSync } = require("fs");
+const { resolve } = require("path");
+
+const webAPIUrl = "https://developer.mozilla.org/en-US/docs/Web/API";
+const sitemapUrl =
+    "https://developer.mozilla.org/sitemaps/en-us/sitemap.xml.gz";
+
+// this is certainly not exhaustive, yet
+const subdirsToFilter = [
```

Comment:
Hmm.. yeah... not exhaustive, and it appears also that we're getting a fair number of URLs which describe non-API members. This isn't a huge deal, since it's unlikely that someone will try to link to them, but `ANGLE_instanced_arrays` for instance really ought not be resolved. There's also the issue that with https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/abort_static, the member is named `abort`, and is a static method...

I think I want to investigate a smarter way of doing this before merging, I happen to know someone fairly active in maintaining MDN, so I'm going to ask them if they have any ideas. If not, this is certainly an improvement over today!

Reply:
I was pointed at https://github.com/mdn/browser-compat-data, which looks very promising, will have more time to look at it this weekend if you don't get to it before me :)

Reply:
Agreed, this looks like a very elegant possible solution. I'm unlikely to have time to dig in until the weekend, but if you don't find time before then I'll get on it